core/domain/stats_domain_test.py

```diff
@@ -145,31 +145,148 @@ def test_validate(self):
                 'Home': state_stats_dict
             }
         }
+
         exploration_stats = self._get_exploration_stats_from_dict(
             exploration_stats_dict)
         exploration_stats.validate()
 
-        # Make the exp_id integer.
+    def test_validate_with_int_exp_id(self):
+        state_stats_dict = {
```

Comment:
Here and in the other file, you might consider following the strategy of the initial test which defines the "valid" object as a property of `self`. Then you don't have to repeat this setup every time.

Reply:
Done